src/app/shared/object-grid/admin-search-result-grid-element/collection-search-result/collection-admin-search-result-grid-element.component.html

```diff
@@ -0,0 +1,13 @@
+<ds-collection-search-result-grid-element [object]="object"
```

Comment:
This will be a "hidden" feature while testing it since the discovery configuration only returns items. To test this, the discovery configuration will need to be adjusted to remove the item filter (similar for the communities below)
This is not something to adjust (it's good to have this available)